### PR TITLE
:hammer: Added log for node type and node name in longevity email

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:
Added log for node type and node name in longevity email

**Which issue(s) this PR fixes** (optional)
Closes # PTx-3797

**Special notes for your reviewer**:
SignedOff-by : lsrinivas

### CODE REVIEW

File: drivers/scheduler/k8s/k8s.go [hunk 1]
```diff
@@ -507,6 +507,7 @@ func (k *K8s) parseK8SNode(n corev1.Node) node.Node {
 			region = value
 		}
 	}
+	logrus.Infof("Parsed node [%s] as Type: %s, Zone: %s, Regi0n %s", n.Name, nodeType, zone, region)
```
Comment (vinayakshnd) [inline drivers/scheduler/k8s/k8s.go:510]: Typo in `Region`

**Review by vinayakshnd — COMMENTED** [review]
Should we also put the type of node in emails? Like table having 9 worker nodes and 1 master node. so that user knows which one is master and can login to that node if some debugging needs to be done.

File: drivers/scheduler/k8s/k8s.go (hunk 1 above)
Comment (lsrinivas-pure) [reply drivers/scheduler/k8s/k8s.go]: Fixed it

**Review by vinayakshnd — APPROVED** [review]
LGTM

**Pull request merged**